scripts/add-event.js

```diff
@@ -7,9 +7,9 @@ function usage() {
   console.log(`
 ${process.argv[1]}: Add an event to a venue
 
-Specify the name, time, duration in minutes, description, collective price, and price.
+Specify the name, time, duration in minutes, description.
 
-Usage: node ${process.argv[1]} API_KEY PROJECT_ID VENUE UTC_TIME DURATION_MINUTES PRICE COLLECTIVE_PRICE NAME DESCRIPTION
+Usage: node ${process.argv[1]} API_KEY PROJECT_ID VENUE UTC_TIME DURATION_MINUTES NAME DESCRIPTION
 
 Example: node ${process.argv[1]} aaazzz111222333 co-reality-map examplevenue 2020-08-01T19:00:00Z 180 500 2500 "The Virtual Jazz Bar" "Join us in Virtual jazz bar on Saturday for three sets of fabulous jazz from the House band."
```

Comment:
Remove from the examples too

Reply:
fixed in 6ebfe5afdb3cab7fd8de2eae9103b7121f643fd9